qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -183,27 +197,41 @@ def set_splash(self, msg=None):
     def create_menus(self):
         self.file_menu = self.menuBar().addMenu("&File")
         self.editor_menu = self.menuBar().addMenu("&Editor")
+        self.view_menu = self.menuBar().addMenu("&View")
 
     def create_actions(self):
         # --- general application menu options --
         # file menu
         action_open = create_action(self, "Open File",
                                     on_triggered=self.open_file,
                                     shortcut="Ctrl+O",
-                                    shortcut_context=Qt.ApplicationShortcut)
+                                    shortcut_context=Qt.ApplicationShortcut,
+                                    icon_name="fa.folder-open")
         action_save = create_action(self, "Save File",
                                     on_triggered=self.save_file,
                                     shortcut="Ctrl+S",
-                                    shortcut_context=Qt.ApplicationShortcut)
+                                    shortcut_context=Qt.ApplicationShortcut,
+                                    icon_name="fa.save")
+        action_manage_directories = create_action(self, "Manage User Directories",
+                                                  on_triggered=self.open_manage_directories,
+                                                  icon_name="fa.folder")
 
         action_quit = create_action(self, "&Quit", on_triggered=self.close,
                                     shortcut="Ctrl+Q",
                                     shortcut_context=Qt.ApplicationShortcut)
```

Comment:
How about adding [`power-off`](https://fontawesome.com/icons/power-off?style=solid) for Quit?